cmd/examples/posix-oneshot/README.md

```diff
@@ -17,6 +17,10 @@ echo "foo" > /tmp/stuff/foo
 echo "bar" > /tmp/stuff/bar
 echo "baz" > /tmp/stuff/baz
 
+# Make a blank dir where we'll store the log data
+export LOG_DIR="/tmp/tlog"
+mkdir "$LOG_DIR"
```

Comment:
`mkdir` is not required as the `posix-oneshot` binary will initialize the directory.

```sh
➜  tessera git:(main) go run ./cmd/examples/posix-oneshot --storage_dir=${LOG_DIR} --entries="/tmp/stuff/*"
I1005 20:39:03.655774 1441449 files.go:516] Initializing directory for POSIX log at "/tmp/tlog" (this should only happen ONCE per log!)
I1005 20:39:03.857020 1441449 main.go:121] 0: /tmp/stuff/bar
I1005 20:39:03.857050 1441449 main.go:121] 1: /tmp/stuff/baz
I1005 20:39:03.857055 1441449 main.go:121] 2: /tmp/stuff/foo
```